src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    addBook: (state, action) => ({
+      ...state,
+      books: [...state.books, action.payload],
+    }),
```

Comment:
- Good job done so far, but there are some issues with some parts of your projects, please note that the `addBook` action in the reducer should be used to mutate the initial state not return a new different state. 
- Since your initial state is an array `line 9` of your code will return an error because your initial state does not have an object containing a `books` key. Please fix this 🙏